.github/workflows/build-images-base.yaml

```diff
@@ -23,6 +23,14 @@ jobs:
     environment: release-base-images
     runs-on: ubuntu-20.04
     steps:
+      - name: Checkout master branch to access local actions
+        uses: actions/checkout@2541b1294d2704b0964813337f33b291d3f8596b
+        with:
+          ref: ${{ github.event.repository.default_branch }}
+          persist-credentials: false
+      - name: Set Environment Variables
+        uses: ./.github/actions/set-env-variables
+
```

Comment:
Actually I'm thinking probably we'd be better off using the very new repo configuration variables instead => https://github.blog/changelog/2023-01-10-github-actions-support-for-configuration-variables-in-workflows/

Reply:
oh this is super cool, i'll work with joe today to add configuration variables for these 🎉

Reply:
https://github.com/cilium/cilium/pull/23145 🥳

Reply:
cool! I'll close this then